Dockerfile

```diff
@@ -22,10 +22,19 @@ RUN chmod +x /usr/local/bin/dockerize
 ARG GIT_COMMIT_ID=unspecified
 ENV GIT_COMMIT_ID=$GIT_COMMIT_ID
 
+# Target frequencies needed. Make them true from docker compose. 
+
+ENV SCHEDULE_BACKUP_HOURLY "false"
```

Comment:
@vijayabharathib can we set this as true by default?

Reply:
done.